src/dpq2/oids.d

```diff
@@ -285,7 +299,7 @@ enum OidType : Oid
     TimeStampWithZoneArray = 1185,
     TimeIntervalArray = 1187,
     NumericArray = 1231,
-    TimeWithZoneArray = 1270, 
+    TimeWithZoneArray = 1270,
```

Comment:
Also should be added into `isSupportedArray`

Reply:
added